src/model/station.js

```diff
@@ -0,0 +1,46 @@
+export function getStations() {
+  return JSON.parse(localStorage.getItem('stations'));
+}
+
+export function addStation(newStation) {
+  let oldStations = localStorage.getItem('stations');
+  if (oldStations === null) {
+    localStorage.setItem('stations', JSON.stringify([newStation]));
+    return true;
+  } else {
+    let stations = JSON.parse(oldStations);
+    if (stations.includes(newStation) === false) {
+      stations.push(newStation);
+      localStorage.setItem('stations', JSON.stringify(stations));
+      return true;
+    }
+    return false;
+  }
+}
+
+function checkLineIncludedStation(lines, station) {
+  let ret = lines.every((line) => {
+    if (
+      line.stations.filter((item) => item !== station).length !=
+      line.stations.length
+    ) {
+      return false;
+    } else {
+      return true;
+    }
+  });
+  return ret;
+}
+
+export function deleteStation(station) {
+  let oldStations = JSON.parse(localStorage.getItem('stations'));
+  let lines = JSON.parse(localStorage.getItem('lines'));
+
+  if (checkLineIncludedStation(lines, station) === false) {
+    return false;
+  } else {
+    const stations = oldStations.filter((element) => element != station);
+    localStorage.setItem('stations', JSON.stringify(stations));
+    return true;
+  }
+}
```

Comment:
```
  let oldStations = JSON.parse(localStorage.getItem('stations'));
  let lines = JSON.parse(localStorage.getItem('lines'));
```
다음과 같이 데이터 정제 부분을 model 로직으로 뺀다면 조금 더 로직을 단순화 시킬 수 있을것같아서 리뷰했습니다. 사실 oldStations와 lines는 함수 내에서 로직이 복잡하지않고 한번밖에 쓰이지 않기때문에 변수에 할당하기보다는 바로 명시해주는게 좋다고 생각합니다. 

또한, 함수 최상단에 return이 없기때문에 로직에는 문제가 없을지라도 가독성 측면에서 많이 떨어진다고 생각합니다. 

checkLineIncludedStation 같은 경우 함수 이름내에 충분히 기능이 내포돼있으므로 `checkLineIncludedStation(lines, station) === false` 보다는 `!checkLineIncludedStation(lines, station)` 가 가독성 측면에서 조금 더 좋다고 생각해서 리뷰했습니다.

```suggestion
export function deleteStation(station) {
  if (!checkLineIncludedStation(lines, station)) {
    return false;
  } 
  localStorage.setItem('stations', JSON.stringify(oldStations.filter((v) => v != station)));
  return true;
}
```